src/components/Form/NumberInput/NumberInput.test.js

```diff
@@ -0,0 +1,33 @@
+import NumberInput from './NumberInput'
```

Comment:
Ah nice I forgot to do this. I will add some unit tests for the other Form components too 👍